lib/generators/krikri/templates/catalog_controller.rb

```diff
@@ -66,22 +66,7 @@ class CatalogController < ApplicationController
 
     config.index.thumbnail_field = :preview_id
 
-    # TODO: Decide whether we want the show (single result) view metadata
-    # to come from the search index or from marmotta
-    #
-    # solr fields to be displayed in the show (single result) view
-    #   The ordering of the field names is the order of the display
-    config.add_show_field 'sourceResource_title', :label => 'Title'
-    config.add_show_field 'sourceResource_description', :label => 'Description'
-    config.add_show_field 'sourceResource_date_providedLabel', :label => 'Date'
-    config.add_show_field 'sourceResource_type_id', :label => 'Type'
-    config.add_show_field 'sourceResource_format', :label => 'Format'
-    config.add_show_field 'sourceResource_rights', :label => 'Rights'
-    config.add_show_field 'dataProvider_name', :label => 'Data Provider'
-    config.add_show_field 'sourceResource_creator_name', :label => 'Creator'
-    config.add_show_field 'sourceResource_spatial_name', :label => 'Place'
-    config.add_show_field 'sourceResource_subject_name', :label => 'Subject'
-    config.add_show_field('sourceResource_collection_title',
-                          :label => 'Collection')
+    # This is not working:
+    # config.show.route = { controller: 'krikri/comparisons' }
```

Comment:
What I'm trying to do here (the above commented-out line) is to get each item on `catalog#index` to link to `krikri/comparisons#show` (the side-by-side comparison of original and enhanced record).  I successfully did something similar to link each item from `krikri/validation_reports#index` to `krikri/comparisons#show` (see https://github.com/dpla/KriKri/pull/21/files#diff-7e5a17e15135a492624373e1c7301358R47).  However, in this case, linking from a route outside of the `krikri` namespace to one inside the `krikri` namespace seems to be stumping me.  With the code that is commented out here, I get this error message: 

`No route matches {:action=>"show", :controller=>"krikri/comparisons", :id=>#<SolrDocument:0x007f85bc67e8c0...`

I don't know that this is possible, given that Blacklight is using this `:controller` style definition (see http://www.mutuallyhuman.com/blog/2011/09/22/rails-3-1-engine-namespaces-can-creep-into-urls-in-application-layout/).